cmd/influx/main.go

```diff
@@ -301,6 +301,10 @@ func writeConfigToPath(tok, org, path, dir string) error {
 }
 
 func checkSetup(host string, skipVerify bool) error {
+	if host == "-" {
```

Comment:
Why is this necessary here?

Reply:
There is a new *--dry-run* flag in "influx write" command that helps to tune CSV to line protocol conversion, it prints lines to stdout. This dry-run does not require any check of influx server setup.

Reply:
have you considered making dry run a new command instead?

```sh
influx write dryrun
```

Reply:
Thank you, I only wanted to bypass the 'setup' middleware for dry-run, and this solution does not appear to me on the first sight. Will do.

Reply:
did the dryrun command get added? I'm not seeing it 😬